docs/user-guide/mvd-configuration.md

```diff
@@ -292,7 +292,90 @@ cp "//'[output_dataset_name]'" 'zlux-app-server/deploy/instance/ZLUX/serverConfi
 }
 ```
 
-### Enabling tracing
+## Applying role-based access control to application services
+
+To use role-based access control (RBAC) for Zowe service endpoints, enable RBAC for Zowe, and then use a z/OS security product such as RACF to map roles and authorities to Zowe service endpoints.
+
+After you configure RBAC, when a Zowe application wants to access an endpoint the application server sends the current user ID and endpoint identifier URL to the ZSS server. The ZSS server queries the z/OS System Authorization Facility (SAF), which determines whether the access is authorized.
```

Comment:
"sends [...] and endpoint identifier URL to the ZSS server" -- this is not exactly correct. Not sure if the doc needs to be that detailed. I would say something like "the application server communicates with the ZSS server to find out if the current user ID  can access the endpoint identifier URL"